hack/local-up-cluster.sh

```diff
@@ -77,6 +77,11 @@ export KUBE_CACHE_MUTATION_DETECTOR
 # START_MODE can be 'all', 'kubeletonly', or 'nokubelet'
 START_MODE=${START_MODE:-"all"}
 
+# By default, we ensure all resources associated with a pod are cleaned up prior to deletion.
+# If you need to debug a pod's volume information, and wish for it not to be deleted during
+# development, set this flag to true, otherwise leave untouched.'
+KEEP_TERMINATED_POD_VOLUMES=${KEEP_TERMINATED_POD_VOLUMES:-false}
```

Comment:
lgtm, but this will change the previous default behaviour and should make all the developers aware this to avoid confusion.